.github/workflows/build.yml

```diff
@@ -78,6 +78,7 @@ jobs:
         image: [
           { base: 'quay.io/centos/centos:stream8', tag: 'centos-8', extra: '', unavailable: 'samba' },
           { base: 'quay.io/centos/centos:stream9', tag: 'centos-9', extra: 'centos-latest', unavailable: 'samba' },
```

Comment:
remove centos-latest

Reply:
Removed.

Reply:
I meant on the line 80 (from centos-9, since you added the tag to centos-10). But I guess both variants work since 10 is still in early development.